src/utils/handleUserConfig.ts

```diff
@@ -173,9 +173,7 @@ export const handleInitData = (
             config.stackDetails.environment;
 
         config.stackDetails.branch =
-            initData.stackDetails?.branch ??
-            config.stackDetails.branch ??
-            "main";
+            initData.stackDetails?.branch ?? config.stackDetails.branch;
```

Comment:
why are we removing default main branch?

Reply:
we can always have `?branch=main`, right?

Reply:
If the customer is not providing the branch in stackDetails object currently we were forcing it to open in the main branch instead of this we should open the Contentstack in whatever the current branch is set in the contentstack cookie